spec/v1/providers/auth.spec.ts

```diff
@@ -135,6 +143,190 @@ describe('Auth Functions', () => {
       });
     });
 
+    describe('beforeCreate', () => {
+      it('should create the function without options', () => {
+        wrapHandlerStub.returns(() => Promise.resolve());
```

Comment:
q: Why is this stubbing needed for test🤔 ?

We are never calling the wrapped `fn` and I thought `wrapHandler` doesn't do anything "side-effect"-y unless the wrapped function is called.

Reply:
Ah you're right, when I had the custom verification I didn't want hack it so I put a stub. Removed it in the latest commit